### ISSUE TITLE
[feature request] support the coverage to 2 decimal

### ISSUE THREAD

**Issue by adamxchen** [form-com/diff-coverage-gradle#74]
**Describe the feature**
Since this plugin is an extension of Jacoco, and Jacoco allows us to configure the coverage to multiple decimals (e.g. 0.65), I had the wrong impression that we can do the same for this plugin.

When I set the `minBranches` to 0.61, the exception still reports the same message
```
java.lang.Exception: Rule violated for bundle clients: branches covered ratio is 0.2, but expected minimum is 0.6
```

**Desktop (please complete the following information):**
 - OS: macOS 12.6.2
 - Gradle version: 6.9.3
 - Diff Coverage plugin version 0.9.5

**To Reproduce**
```groovy
diffCoverageReport {
    diffSource.file = 'diff.patch'

    violationRules {
      minBranches = 0.61
      failOnViolation = true
    }
}
```

**Expected behavior**
The report can be more precise when the coverage is set to 2 decimals, such as
```
java.lang.Exception: Rule violated for bundle clients: branches covered ratio is 0.29, but expected minimum is 0.61
```

**Additional context**
I checked the [config usage](https://github.com/form-com/diff-coverage-gradle/search?q=minBranches) and seems we only set a single decimal for the `double` type, which might be the reason but I could be wrong.
Please let me know if it can be supported. Thanks!

**Comment by SurpSG** [comment]
Hi @adamxchen 

Seems, when you set min coverage to `0.x` then jacoco ceils coverage ratio to the same format `0.x`.

However, when you set coverage to `0.xx` then jacoco computes the coverage ratio with the same precision.

I've tried to run the plugin with different gradle versions:
```
 5.0
 5.3
 6.7.1
 6.9.3
 7.4.2
 7.6
 8.0-rc-1
```

And configuration:
```
 minBranches = 0.66
 minLines = 0.77
 minInstructions = 0.88
 failOnViolation = true
```
And I've received actual error messages that are equal to expected:
```
instructions covered ratio is 0.52, but expected minimum is 0.88
branches covered ratio is 0.50, but expected minimum is 0.66
lines covered ratio is 0.66, but expected minimum is 0.77
```
Diff-Coverage version: `0.9.5`

In other words, I wasn't able to reproduce the issue.

Could you please provide a sample project in which we could observe the issue?

**Comment by adamxchen** [comment]
Uh.. I think that's my mistake. i used 0.60 and hope it will convert it to `0.xx` for me however, 0.60 is treated as 0.6 which could be the reason.

Apologies for it. I'm going to close this issue! Thank you!
